legacy/firmware/usb.h

```diff
@@ -25,8 +25,32 @@
 void usbInit(void);
 void usbPoll(void);
 void usbReconnect(void);
+
+/*
+ * Setting this value to 0 will limit the protobuf messages `usbPoll` and
```

Comment:
I think the comment should be extended since the protobuf USB messages are just one of three types of packets that are affected by this.

Reply:
Updated comment to reflect those messages.